src/app/@types/invoice.ts

```diff
@@ -1,16 +1,16 @@
-import { BillPart } from '../shared/billPart.model';
+import { InvoicePart } from '../shared/invoice-part.model';
```

Comment:
"InvoicePart" is not the most obvious name. "Company" would ne more straightforward